package/yast2-firewall.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-firewall
-Version:        4.1.11
+Version:        4.2.0
```

Comment:
This is the very same change as in the SP1 branch, right? Then we can keep the SP1 version, it should be changed only when doing TW or SP2 change not present in SP1...

Reply:
but I modify spec, so spec is not same and I think in that case if htere is differ, we need to have different version, not?